src/SFA.DAS.Roatp.CourseManagement/Application/Standards/Queries/GetProviderCourse/GetProviderCourseQueryHandler.cs

```diff
@@ -41,6 +43,27 @@ public async Task<GetProviderCourseResult> Handle(GetProviderCourse.GetProviderC
                 return null;
             }
 
+            var providerCourseLocations = await _courseManagementApiClient.Get<List<GetProviderCourseLocationsResponse>>(new GetProviderCourseLocationsRequest(request.Ukprn, request.LarsCode));
```

Comment:
We're trying to use a different approach with calling information from the inner apis.   We're using GetWithResponseCode, and if there is we get a response code we don't expect, we just pass the response code back.r
I will double-check with Deven, and will reply here about it